ring/gallery.html

```diff
@@ -28,9 +29,23 @@
             <br>
             <br>
             <span class="c1">To view an image, click on its thumbnail.
+            </span>
             <br>
             <br>
-            <a href="1.jpg"><img src="1small.jpg" height="107" width="66" alt="no alt text"></a> <a href="2.jpg"><img src="2small.jpg" height="104" width="69" alt="no alt text"></a> <a href="3.jpg"><img src="3small.jpg" height="78" width="98" alt="no alt text"></a> <a href="4.jpg"><img src="4small.jpg" height="104" width="69" alt="no alt text"></a> <a href="5.jpg"><img src="5small.jpg" height="89" width="109" alt="no alt text"></a> <a href="6.jpg"><img src="6small.jpg" height="120" width="72" alt="no alt text"></a></span>
+            <a href="1.jpg"><img src="1small.jpg" height="107" width="66" alt="no alt text"></a>
```

Comment:
These images are not very descriptively named. (1.jpg through 6.jpg? Really?) You should fix that.